tests/integration-tests/configs/common_tests

```diff
@@ -0,0 +1,507 @@
+cfn-init:
```

Comment:
can we add .yaml to the file?

Reply:
done, but I had to move the `common.yaml` under a `common` folder to avoid the validation performed by the config_validator